secp256k1-sys/depend/secp256k1-HEAD-revision.txt

```diff
@@ -1,2 +0,0 @@
-# This file was automatically created by ./vendor-libsecp.sh
```

Comment:
Probably a dumb question but we are doing this to use the mimblewimble specific libs here?
And it replaces all the removed vendored code below?